src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -989,14 +1001,15 @@ def _json_pretty_print(self, content):
                 ': '))
 
     def _utf8_urlencode(self, data):
+        return data
```

Comment:
This method is just disabled as it is. The data will not be urlencoded so its basically broken.